src/Microsoft.DotNet.CodeFormatter.Analyzers/NameHelper.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.CodeFormatter.Analyzers
+{
+    /// <summary>
+    /// Commonly used name functions
+    /// </summary>
+    internal static class NameHelper
+    {
+        /// <summary>
+        /// Get the full name of a symbol. So a field might look like
+        /// "OuterNamespace.Inner.ClassName.FieldName"
+        /// </summary>
+        /// <param name="symbol">symbol to get name of</param>
+        /// <returns>Full display name for a symbol</returns>
+        internal static string GetFullName(ISymbol symbol)
```

Comment:
Ya one can use `symbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)` to get the full name. I'll merge this is in now since you just moved the file.